torch/_jit_internal.py

```diff
@@ -926,6 +927,11 @@ def _qualified_name(obj):
     #     raise RuntimeError(f"Could not get qualified name for class '{name}': "
     #                        f"the attr {name} on module {module_name} is not the the class")
 
+    # torch.package and TorchScript have separate mangling schemes to avoid
+    # name collisions from multiple packages. To avoid them interfering with
+    # each other, remove the package mangling here.
+    module_name = package_mangling.demangle(module_name)
```

Comment:
I'm assuming this is a no-op when `module_name` has not been mangled by package?

Reply:
correct